torch/nn/parallel/parallel_apply.py

```diff
@@ -8,31 +8,37 @@
     import Queue as queue
 
 
-def parallel_apply(modules, inputs):
+def parallel_apply(modules, inputs, kwargs_tup=None):
     assert len(modules) == len(inputs)
+    if kwargs_tup:
+        assert len(modules) == len(kwargs_tup)
+    else:
+        kwargs_tup = tuple({} for gpu in modules)
     # Fast track
     if len(modules) == 1:
-        return (modules[0](*inputs[0]),)
+        return (wrap(modules[0], *inputs[0], **kwargs_tup[0]), )
```

Comment:
Remnant from the refactor, no idea how that got through and I didn't catch it on tests. Fixed.